WordPress/Resources/en.lproj/Localizable.strings

```diff
@@ -7167,12 +7220,39 @@
 /* Displayed in the Stats widgets when there is no network */
 "Stats will be updated next time you're online" = "Stats will be updated next time you're online";
 
+/* Title for Comments count in Latest Post Summary stats card. */
+"stats.insights.latestPostSummary.comments" = "Comments";
+
+/* Title for Likes count in Latest Post Summary stats card. */
+"stats.insights.latestPostSummary.likes" = "Likes";
+
+/* Publish date of a post displayed in Stats. Placeholder will be replaced with a localized relative time, e.g. 2 days ago */
+"stats.insights.latestPostSummary.publishDate" = "Published %@";
+
+/* Title for Views count in Latest Post Summary stats card. */
+"stats.insights.latestPostSummary.views" = "Views";
+
+/* Insights 'Most Popular Time' header. Fire emoji should remain part of the string. */
+"stats.insights.mostPopularCard.title" = "🔥 Most Popular Time";
+
+/* Label showing the percentage of views to a user's site which fall on a particular day. */
+"stats.insights.mostPopularCard.viewPercentage" = "%d%% of views";
```

Comment:
Love the reverse DNS localization keys 👍

Reply:
![whoop whoop](https://media.giphy.com/media/1ThndC5odGuUU/giphy.gif)